sw/airborne/firmwares/rotorcraft/autopilot_firmware.c

```diff
@@ -92,7 +93,13 @@ bool WEAK autopilot_ground_detection(void) {
 bool WEAK autopilot_in_flight_end_detection(bool motors_on UNUSED) {
   if (autopilot_in_flight_counter > 0) {
     /* probably in_flight if thrust, speed and accel above IN_FLIGHT_MIN thresholds */
-    if ((stabilization.cmd[COMMAND_THRUST] <= AUTOPILOT_IN_FLIGHT_MIN_THRUST) &&
+    /** Select the correct available Thrust Setting*/
+#ifdef COMMAND_THRUST
+    float thrust_level = commands[COMMAND_THRUST];
+#else
+    float thrust_level = stabilization.cmd[COMMAND_THRUST];
```

Comment:
I don't get this. If `COMMAND_THRUST` is not defined, the `else` case will fail.

Reply:
indeed this is wrong, i corrected it